src/eos-companion-app-integration-helper.c

```diff
@@ -506,10 +510,8 @@ list_application_infos (GCancellable  *cancellable,
           if (!is_compatible_app)
             continue;
 
-          if (!load_desktop_info_key_file_for_app_id (app_name, &app_info, error))
-            return NULL;
+          app_info = load_desktop_info_key_file_for_app_id (app_name, &check_app_error);
```

Comment:
I don't believe `check_app_error` exists by the time it gets introduced with this commit.

Reply:
This should also have been fixed now